sonic-xcvrd/src/xcvrd_utilities/y_cable_helper.py

```diff
@@ -0,0 +1,510 @@
+                return 2
+            elif status == "standby":
+                y_cable.toggle_mux_to_torA(physical_port)
+                return 1
+
+        ''' TODO: Now that mux has been issued a command for toggle, check to see if
+            mux has indeed been toggled
+            This check might not be neccessary
+            active_side = y_cable.check_active_linked_tor_side(physical_port)
+        '''
+    else:
+        '''
+           Y cable ports should always have
+           one to one mapping of physical-to-logical
+           This should not happen'''
+        helper_logger.log_warning(
+            "Error: Retreived multiple ports for a Y cable table".format(logical_port_name))
+        return -1
+
+
+def update_appdb_port_mux_cable_response_table(logical_port_name, asic_index, appl_db):
+    physical_port_list = logical_port_name_to_physical_port_list(
+        logical_port_name)
+
+    if len(physical_port_list) == 1:
+
+        physical_port = physical_port_list[0]
+        if _wrapper_get_presence(physical_port):
+            status = None
+            y_cable_response_tbl = {}
+            read_side = y_cable.check_read_side(physical_port)
+            y_cable_response_tbl[asic_index] = swsscommon.Table(
+                appl_db[asic_index], swsscommon.APP_MUX_CABLE_RESPONSE_TABLE_NAME)
+
+            if not read_side:
+                status = 'failure'
+
+                fvs = swsscommon.FieldValuePairs([('status', status)])
+                y_cable_response_tbl[asic_index].set(logical_port_name, fvs)
+
+                helper_logger.log_warning(
+                    "Error: Could not get read side for mux cable port probe failed {}".format(logical_port_name))
+                return
+
+            active_side = y_cable.check_active_linked_tor_side(physical_port)
+            if not active_side:
+                status = 'failure'
+
+                fvs = swsscommon.FieldValuePairs([('status', status)])
+                y_cable_response_tbl[asic_index].set(logical_port_name, fvs)
+
+                helper_logger.log_warning(
+                    "Error: Could not get active side for mux cable port probe failed {}".format(logical_port_name))
+                return
+
+            if read_side == active_side and (active_side == 1 or active_side == 2):
+                status = 'active'
+            elif read_side != active_side and (active_side == 1 or active_side == 2):
+                status = 'standby'
+            else:
+                status = 'inactive'
+
+
+            fvs = swsscommon.FieldValuePairs([('status', status)])
+
+            y_cable_response_tbl[asic_index].set(logical_port_name, fvs)
+        else:
+            helper_logger.log_warning(
+                "Error: Could not establish presence for  Y cable port {}".format(logical_port_name))
+    else:
+        '''
+        Y cable ports should always have
+        one to one mapping of physical-to-logical
+        This should not happen'''
```

Comment:
Suggest using comments (`#`) here instead of docstring. Docstrings are intended to document how to _use_ the code, whereas comments are indented to document how the code _works_.

Reply:
fixed